pkg/operation/botanist/etcdencryption.go

```diff
@@ -164,16 +260,36 @@ func (b *Botanist) RewriteShootSecretsIfEncryptionConfigurationChanged(ctx conte
 		return b.CheckSums[common.EtcdEncryptionSecretName]
 	}()
 
-	if err := b.rewriteShootSecretsIfChecksumChanged(ctx, checksum); err != nil {
```

Comment:
why was this changed?

Reply:
Shouldn't have changed, something I missed during a rebase I guess.